frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTask.component.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="sidebar"></div>
+</template>
```

Comment:
Same, maybe we can create a feedback-task folder sibling to the other task (text-classifier) and put the components inside (to keep in commons only common components)

Reply:
yes. This branch is only for the questionnaire but I put this div just to have a global view. When the sidebar development will start, it will be in a new branch.